### PR TITLE
FEAT: auto copy about text to clipboard

### PR DESCRIPTION
**Pull request by hiiamboris**
imagine how often I'm doing this manually ;)

### CODE REVIEW

**Comment by greggirwin** [comment]
My instinct is that it should be done with a refinement (e.g. `about/cc`). It's absolutely a common use case for us, but when apps copy to the clipboard without my knowledge it can be very frustrating. Almost as much as when I overwrite it myself and have already closed what I had copied.

**Review by greggirwin — COMMENTED** [review]
See comment about maybe using a refinement for this new behavior.

File: environment/console/help.red [hunk 1]
```diff
@@ -603,6 +603,7 @@ help-ctx: context [
 	set 'about func [
 		"Print Red version information"
 		/debug "Print full Red and OS version information suitable for submitting issues"
+		/cc "Also Copy the result into Clipboard"
```
Comment (greggirwin) [inline environment/console/help.red:606]: Small thing but let's lose the caps mid sentence: "Also copy the result into clipboard"

Comment (hiiamboris) [reply environment/console/help.red:606]: Then the name cc become a bit cryptic

Comment (greggirwin) [reply environment/console/help.red:606]: Ah, I see how you intended it. Being old, I interpret it as Carbon Copy, as it's used with email.

Comment (greggirwin) [reply environment/console/help.red:606]: If we do `/cc "Also copy to clipboard"` it puts the 2 c's closer together. Then the case also matches the refinement case.

Comment (greggirwin) [reply environment/console/help.red:606]: And, No, `/c2c` is not acceptable. ;^)

Comment (hiiamboris) [reply environment/console/help.red:606]: :D

**Comment by greggirwin** [comment]
Good work. Thanks!

**Comment by greggirwin** [comment]
Please announce this in Gitter, and we should update the issue template to note it.

**Pull request merged**